### DISCUSSION TITLE
Scheduling Function

### DISCUSSION THREAD

**Discussion by mehdisfh** [contiki-ng/contiki-ng#2461] [Q&A] [answered]
Hi everyone,

I would like to bring your attention to the simple scheduling function provided by Contiki-ng in the Sixtop example. Currently, it lacks a bandwidth monitoring mechanism as well as a bandwidth adaptation mechanism. I'm curious to know if anyone is aware of a complete scheduling function that has been implemented or introduced for Contiki-ng. By "complete," I mean a scheduling function that supports all the behaviors defined in RFC 9033 (6TiSCH Minimal Scheduling Function (MSF)).

Thank you in advance for your answers.

**Comment by arurke** [comment, marked as answer]
Hi, sorry for the late reply, have you had a look at #1000 ?
